static/app/utils/formatters.spec.tsx

```diff
@@ -14,7 +14,9 @@ import {
 
 describe('getDuration()', function () {
   it('should format durations', function () {
-    expect(getDuration(0, 2)).toBe('0.00ms');
+    expect(formatSecondsToClock(0)).toBe('00:00');
+    expect(formatSecondsToClock(0.001)).toBe('00:00.001');
+    expect(formatSecondsToClock(0.01)).toBe('00:00.010');
```

Comment:
is this the right test to add these to? maybe I don't know the relation between `formatSecondsToClock` and `getDuration`

Reply:
`getDuration` is more sentence style, it's returning something like `2 minutes` 

all these formatters are built very differently, with different options available. Its probably ripe for some new base classes/fns.